data/bunrodea/bunrodea missions.txt

```diff
@@ -0,0 +1,216 @@
+# Copyright (c) 2023 by Amazinite
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "First Contact: Bunrodea"
+	landing
+	source
+		near "Sol" 0 100
+		not attributes "urban"
+		attributes "spaceport"
+	destination "Erabuthro"
+	clearance "Unlike before, you don't receive any hostility entering this region. After hailing the planet that the map pointed you towards, you hear an alien voice respond. Before you can think of what to say, the small device that the aliens threw at you on <origin> responds for you. The voice on the other end says something short, then cuts the feed. It seems like you've been cleared to land."
```

Comment:
![image](https://user-images.githubusercontent.com/29336153/218914564-d21ee198-74a3-4aff-a5ea-2bb7314a3de8.png)

This text exceeds the planetary information box.